Riot.xcodeproj/xcshareddata/xcschemes/Riot.xcscheme

```diff
@@ -15,7 +16,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "57B13CC4586E9D43ED24DE1E"
-               BuildableName = "Element.app"
+               BuildableName = "Riot.app"
```

Comment:
@langleyd I've seen also that this has been changed, should I keep your change, or keep the original one?